components/widgets/climate/carbon-flux/selectors.js

```diff
@@ -0,0 +1,273 @@
+                          : x + width - offset
+                      }
+                      y={y}
+                      textAnchor={removalsData > 0 ? 'start' : 'end'}
+                      fill="#000"
+                      fontSize={14}
+                    >
+                      REMOVALS
+                    </text>
+                  </g>
+                );
+              },
+            },
+          },
+          flux: {
+            fill: netFluxData > 0 ? netEmissions : netRemovals,
+            background: false,
+            labelList: {
+              content: (props) => {
+                // eslint-disable-next-line react/prop-types
+                const { x, y, width, height, offset } = props;
+                return (
+                  <g transform={`translate(0 ${height / 2 + offset})`}>
+                    <text
+                      x={
+                        netFluxData > 0
+                          ? x + width + offset
+                          : x + width - offset
+                      }
+                      y={y}
+                      textAnchor={netFluxData > 0 ? 'start' : 'end'}
+                      fill="#000"
+                      fontSize={14}
+                    >
+                      {netFluxData > 0 ? 'NET EMISSIONS' : 'NET REMOVALS'}
+                    </text>
+                  </g>
+                );
+              },
+            },
+          },
+        },
+      },
+      referenceLine: { x: 0, stroke: '#606060', strokeWidth: 2 },
+      tooltip: [
+        {
+          label: 'Carbon flux (per year)',
+        },
+        {
+          key: 'removals',
+          label: 'Removals',
+          unitFormat: (value) => `${format('.3s')(value)}tCO\u2082e`,
+          color: removals,
+        },
+        {
+          key: 'emissions',
+          label: 'Emissions',
+          unitFormat: (value) => `${format('.3s')(value)}tCO\u2082e`,
+          color: emissions,
+        },
+        {
+          key: 'flux',
+          label: netFluxData > 0 ? 'Net emissions' : 'Net removals',
+          unitFormat: (value) => `${format('.3s')(value)}tCO\u2082e`,
+          color: netFluxData > 0 ? netEmissions : netRemovals,
+        },
+      ],
+    };
+  }
+);
+
+export const parseSentence = createSelector(
+  [
+    getData,
+    getSentences,
+    getIndicator,
+    getLocationName,
+    getSettings,
+    getAdminLevel,
+  ],
+  (data, sentences, indicator, locationName, settings, adminLevel) => {
+    if (!data || isEmpty(data)) return null;
+
+    const {
+      globalInitial,
+      globalWithIndicator,
+      initial,
+      withIndicator,
+    } = sentences;
+    const { startYear, endYear } = settings;
+
+    const yearTotal = endYear - startYear + 1;
+    const { emissions, removals, flux } = data[0];
+
+    const params = {
+      indicator: indicator && indicator.label,
+      startYear,
+      endYear,
+      location: locationName === 'global' ? 'the world' : locationName,
```

Comment:
No need for this, right? Since the global sentences are hardcoded with global?

Reply:
totally right, I included it before including the global sentence (my left over)